scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -142,6 +145,7 @@ case class ScalafmtConfig(
     danglingParentheses: Boolean = false,
     poorMansTrailingCommasInConfigStyle: Boolean = false,
     verticalMultilineAtDefinitionSite: Boolean = false,
+    verticalMultilineAtDefinitionSiteArityThreshold: Int = 100,
```

Comment:
I really wanted to make this an Option but there isn't a `ConfDecoder[Option[A]]` defined. Does it make sense to make it an option?

Reply:
I can't remember why I never implemented a decoder for Option[T], there was some reason. I agree dummy Int values are not great, but it's consistent with some other settings so it's fine here

Reply:
Alright, I'll leave this as it for now then 👍